server/network.go

```diff
@@ -284,16 +284,18 @@ var errAborted = errors.New("aborted")
 func (lc *localNetwork) waitForLocalClusterReady(ctx context.Context) error {
 	color.Outf("{{blue}}{{bold}}waiting for all nodes to report healthy...{{/}}\n")
 
-	hc := lc.nw.Healthy(ctx)
-	select {
-	case <-lc.stopc:
-		return errAborted
-	case <-ctx.Done():
-		return ctx.Err()
-	case err := <-hc:
-		if err != nil {
-			return err
+	healthyCtx, healthyCtxCancel := context.WithCancel(ctx)
```

Comment:
I think you do not need this. Healthy is already taking into account stop events. You can ignore lc.stopc.

Reply:
removed